app/views/govuk_publishing_components/components/docs/contextual_footer.yml

```diff
@@ -76,13 +77,14 @@ examples:
       content_item:
         title: "A content item"
         links:
-          topics:
+          mainstream_browse_pages:
           - title: Apprenticeships, 14 to 19 education and training for work
             base_path: /browse/education/find-course
-            document_type: topic
+            document_type: mainstream_browse_page
           - title: Finding a job
             base_path: /browse/working/finding-job
-            document_type: topic
+            document_type: mainstream_browse_page
+          topics:
```

Comment:
I've noticed there are some topics left in the examples. Is that intentional?

Reply:
Not intentional! Thanks, will remove.